@ZachLiss It seems we are having the following error on the CI:

```
** (Mix) You're trying to run :boat_tracker on Elixir v1.13.3 but it has declared in its mix.exs file it supports only Elixir ~> 1.12.3-otp-24
[64](https://github.com/DockYard/boat-tracker/pull/3/checks#step:7:64)
Error: Process completed with exit code 1.
```


Should we downgrade the elixir version on the GitHub actions yaml file?

--- a/.github/workflows/elixir.yml
+++ b/.github/workflows/elixir.yml
@@ -17,7 +17,7 @@ jobs:
     - name: Set up Elixir
       uses: erlef/setup-beam@988e02bfe678367a02564f65ca2e37726dc0268f
       with:
-        elixir-version: '1.13.3'
+        elixir-version: '1.12.3'
         otp-version: '24.3'
     - name: Restore dependencies cache
       uses: actions/cache@v2

